onc-healthit/inferno-program
# FI-1033: Swap HTTP Gem for rest-client

To solve the ongoing HTTPS issues with bulk-data-server, this PR swaps out the `http` gen in `streamed_ndjson_get` for the `rest-client` gem we use elsewhere. This enables us to use the same SSL verification switching code here as elsewhere.

To test: 
* check out this branch
* build it in dockerland: `docker build -t onchealthit/inferno-program:latest .`
* change to the `inferno-site-overlay` directory
* swap out the production certs for the dev certs in `docker-compose.yml`
* start it in inferno-site-overlay: `docker-compose up --build --force-recreate` 
* visit inferno using your IP address
* Verify that `multi patient` tests fail by default, because of the self-signed cert
* change `disable_verify_peer` to `true` in `inferno-config.yml` in `i-s-o`
* restart docker-compose
* verify that the multi-patient tests pass (or skip, due to data missing)

Pull requests into Inferno require the following items to be completed. Submitter and reviewer 
should check the relevant check boxes when the associated item is done. For items that are not 
applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
Requests do not require ticket links.

**Submitter:**
- [x] Internal ticket for this PR: FI-1033
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label


**Reviewer 1:**

Name:
      where appropriate, and accomplishes the task's purpose

**Reviewer 2:**

Name:
      where appropriate, and accomplishes the task's purpose
